demos/middleware/middleware.ts

```diff
@@ -57,6 +57,10 @@ export async function middleware(req: NextRequest) {
     return response
   }
 
+  if (pathname.startsWith('/rewrite-to-edge')) {
```

Comment:
Is this a part of the app dir stuff or is this unrelated?

Reply:
It was so I could test how the changes affected existing sites. I can probably remove it or add it to a different PR